client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -119,6 +123,19 @@ export const useCommandsArrayWpcom = ( {
 		}
 	};
 
+	const downloadSubscribersForSite = async ( slug: string, siteId: number ) => {
```

Comment:
TODO: explore if `window.open` is enough. We wanna just trigger the file download, not navigate.